js/page-specific/swiftype-search-results.js

```diff
@@ -8,13 +8,21 @@ baltimoreCounty.pageSpecific.swiftypeSearchResults = (function swiftypeSearchRes
 
 	function getSearchResults(searchTerm, pageNumber) {
 		var currentPageNumber = pageNumber || 1;
-		var cleanedSearchTerm = searchTerm.replace(/\+/g, '%20');
+		var cleanedSearchTerm = searchTerm;
```

Comment:
Just in case, it might be worth trimming searchTerm. Probably not necessary, but it's a good safeguard.